src/js/components/organisms-page/content-cover/content-cover.html

```diff
@@ -0,0 +1,24 @@
+<article class="content-cover content-cover--cover">
+    <c-image v-if="this.image" v-bind="this.image"></c-image>
+
+    <div class="content-cover__image--overlay"></div>
+
+    <section class="content-cover__container">
+        <div class="flexgrid__cell--xs-6 content-cover-content content-cover--cover-reset">
+             <h3 class="content-cover__title">{{ title }}</h3>
```

Comment:
`v-if description`
description is indicated as required in the component. why is `v-if` used here?